jupyter_packaging/setupbase.py

```diff
@@ -259,12 +259,14 @@ def run(cmd, **kwargs):
     kwargs.setdefault('shell', os.name == 'nt')
     if not isinstance(cmd, (list, tuple)):
         cmd = shlex.split(cmd)
-    cmd_path = which(cmd[0])
-    if not cmd_path:
-        raise ValueError("Aborting. Could not find cmd (%s) in path. "
-                 "If command is not expected to be in user's path, "
-                 "use an absolute path." % cmd[0])
-    cmd[0] = cmd_path
+    if not os.path.isabs(cmd[0]):
```

Comment:
Maybe try `str(cmd[0]).replace(os.sep, “/“)` for the path to test.

Reply:
Could you please explain why should I replace the path separator? `sys.executable` should be an absolute path to the interpreter and `os.path.isabs` should support windows paths.
I don't have an access to windows to debug it.